spam-checker-app/pages/index.js

```diff
@@ -0,0 +1,96 @@
+import axios from "axios";
+import { useState } from "react";
+
+export default function Home() {
+  const [text, setText] = useState("");
+  const [response, setResponse] = useState(null);
+  const [loading, setLoading] = useState(false);
+
+  const exampleText =
```

Comment:
You can set the default value in [the state](https://github.com/RapidAPI/DevRel-Examples-External/pull/283/files#diff-880f0224f81d446f49bbd4acd8aca11113b9305dc984a310a1c9b656fb3c8865R5).